beacon-chain/rpc/endpoints.go

```diff
@@ -558,6 +559,16 @@ func (s *Service) beaconEndpoints(
 			handler: server.PublishBlockV2,
 			methods: []string{http.MethodPost},
 		},
+		{
```

Comment:
since this isn't a beacon-api https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/ this should probably go under prysmBeaconEndpoints ( line 930 and below in the same file) and be under that namespace with the prysm subpath

Reply:
+1

Reply:
Updated